Packs/MicrosoftGraphUser/TestPlaybooks/Microsoft_Graph_User_test.yml

```diff
@@ -545,17 +532,231 @@ tasks:
+      '#none#':
+      - "20"
+    scriptarguments:
+      account_enabled: {}
+      display_name:
+        simple: Graph Test - DELETE
+      mail_nickname:
+        simple: graph
+      on_premises_immutable_id: {}
+      other_properties:
+        simple: jobTitle=Test,city=Tel Aviv
+      password:
+        simple: Aa-123456789Bc!
+      user_principal_name:
+        simple: ${GeneratedUUID}manager@${inputs.baseurl}
+    continueonerror: true
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 2295
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "20":
+    id: "20"
+    taskid: cb1d8651-6efc-4e92-84af-4bc2fc8893f2
+    type: regular
+    task:
+      id: cb1d8651-6efc-4e92-84af-4bc2fc8893f2
+      version: -1
+      name: Get User (manager)
+      description: Retrieve the properties and relationships of user object. [link](https://docs.microsoft.com/en-us/graph/api/user-update?view=graph-rest-1.0)
+      script: '|||msgraph-user-get'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "21"
+    scriptarguments:
+      properties: {}
+      user:
+        simple: ${GeneratedUUID}manager@${inputs.baseurl}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 2470
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "21":
+    id: "21"
+    taskid: a359e85c-2745-49ac-8478-6bf56dfc4a53
+    type: regular
+    task:
+      id: a359e85c-2745-49ac-8478-6bf56dfc4a53
+      version: -1
+      name: Assign test user manager
+      description: 'Assigns a manager to the user. Permissions: - User.ReadWrite.All
+        (Delegated & Application)'
+      script: '|||msgraph-user-assign-manager'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      manager:
+        simple: ${GeneratedUUID}manager@${inputs.baseurl}
+      user:
+        simple: ${GeneratedUUID}test@${inputs.baseurl}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 2645
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "22":
+    id: "22"
+    taskid: 92429cd6-14f4-426d-89da-144566618b92
+    type: regular
+    task:
+      id: 92429cd6-14f4-426d-89da-144566618b92
+      version: -1
+      name: Get test user manager
+      description: Retrieves the properties from the manager of a user
+      script: '|||msgraph-user-get-manager'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "23"
+    scriptarguments:
+      user:
+        simple: ${GeneratedUUID}test@${inputs.baseurl}
```

Comment:
let's verify the expected outputs are returned

Reply:
How do you want to do that exactly?
Just visually with Azure AD next to it and/or adding conditions with operator: isNotEmpty as used in auto generated test playbooks by the demisto-sdk generate-test-playbook command?

Reply:
you can use conditions to see the returned outputs are the expected ones (i.e. the manager is what you set)

you can use any condition/filter you prefer (`Equals` can check the manager returned is the one you set for example)

Reply:
Update the testing playbook for the manager related commands